services/memory/.env.example

```
@@ -23,3 +23,8 @@ IBEX_MEMORY_EMBEDDING_BASE_URL=http://127.0.0.1:8004
 IBEX_MEMORY_MAX_CONTENT_CHARS=10000
 IBEX_MEMORY_PII_REDACT_MIN_CONFIDENCE=0.70
 IBEX_MEMORY_PII_SPACY_MODEL=en_core_web_md
+
+# Dedup stages (m3.C.2 / ADR-0055)
+IBEX_MEMORY_DEDUP_EXACT_ENABLED=true
+IBEX_MEMORY_NEAR_DUPLICATE_SIM_THRESHOLD=0.92
+IBEX_MEMORY_NEAR_DUPLICATE_CANDIDATE_LIMIT=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IBEX_MEMORY_NEAR_DUPLICATE_CANDIDATE_LIMIT key should go before the IBEX_MEMORY_NEAR_DUPLICATE_SIM_THRESHOLD key

(UnorderedKey)